charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -140,6 +164,14 @@ spec:
           env:
             - name: ADDRESS
               value: /var/lib/csi/sockets/pluginproxy/csi.sock
+{{- if .Values.proxy.http_proxy }}
+            - name: HTTP_PROXY
+              value: {{ .Values.proxy.http_proxy | quote }}
+            - name: HTTPS_PROXY
+              value: {{ .Values.proxy.http_proxy | quote }}
```

Comment:
Hmm would it be better to have a different value for HTTPS_PROXY?

Reply:
> Hmm would it be better to have a different value for HTTPS_PROXY?

It depends, in 99% environments is same proxy for both.

Reply:
I think this is a good start. We can do it later if necessary.

Reply:
here is lot of ways, how to do it, from my point of view preffered one:
- name: HTTPS_PROXY
  value: {{ .Value.proxy.https_proxy | default .Value.proxy.http_proxy | quote }}